app.js

```diff
@@ -235,31 +240,52 @@ const tslint = {
       default: 'scss',
       when: answers => {
         return answers.themeFrame === 'None';
-      },
+      }
     },
     {
       type: 'confirm',
       name: 'testing',
       message: 'Do you want Testing? ☢️ ',
-      default: true,
+      default: true
     },
     {
       type: 'confirm',
       name: 'testFrame',
       message:
-        'Do you want to user Jest instead of Karma/Jasmine/Protactor ? ⚙️ ',
+        'Do you want to use Jest instead of Karma/Jasmine/Protactor ? ⚙️ ',
       default: true,
       when: answers => {
         return answers.testing;
-      },
+      }
     },
     {
       type: 'confirm',
       name: 'hammer',
       message: 'Do you want to install HammerJS ? 🔨 ',
-      default: true,
+      default: true
+    },
+    {
+      type: 'confirm',
+      name: 'git',
+      message: 'Do you want to add a git repositorie ? :octocat: ',
+      default: false
     },
+    {
+      type: 'text',
+      name: 'gitLink',
+      message: 'Provide a git repositorie link(ended by .git)  ',
+      when: answers => {
+        return answers.git === true;
```

Comment:
=== true is redundant
the when attribute happens when it returns a true.  answers.git is already a boolean